pkg/config/generate_test.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"github.com/moshebe/gebug/pkg/testutil"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestConfig_Generate(t *testing.T) {
+
+	testutil.FsTest(t, &AppFs, func(t *testing.T) {
```

Comment:
Isn't it will be easier to just call afero.NewFs() at the start of each relevant test instead of this wrap?